hpat/datatypes/hpat_pandas_series_functions.py

```diff
@@ -522,10 +525,10 @@ def hpat_pandas_series_pow_impl(lhs, rhs):
 
         return hpat_pandas_series_pow_impl
 
-    raise TypingError('{} The object must be a pandas.series or scalar. Given rhs: {}'.format(_func_name, rhs))
+    raise TypingError('{} The object must be a pandas.series and argument must be a number. Given rhs: {}'.format(_func_name, rhs))
```

Comment:
Thanks but we still have only `rhs` type printed here. It would be good to add `lhs` type also to see full information